operator/deploy/cr.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apicur.io/v1alpha1
+kind: Apicurito
+metadata:
+  name: apicurito
+spec:
+  replicaCount: 2
```

Comment:
Does need two?

Reply:
No two aren't needed, but it wouldn't hurt.

Apicurito is 100% client only - the server does nothing except serve static html/css/js files.  The great irony is that it scales horizontally **extremely** well....but doesn't need to. :)

Reply:
Well since it is very light I though 2 can enhance high availability at a very low cost, that is why ;)

Reply:
I don't think it's needed at this stage as a default, but I think that this is a nice addition to documentation to show users how they can scale.